Formula/lua.rb

```diff
@@ -80,68 +79,68 @@ def caveats
   end
 
   test do
-    system "#{bin}/lua", "-e", "print ('Ducks are cool')"
+    assert_match "Homebrew is awesome!", shell_output("#{bin}/lua -e \"print ('Homebrew is awesome!')\"")
   end
 end
 
 __END__
 diff --git a/Makefile b/Makefile
-index 7fa91c8..a825198 100644
+index 416f444..1424dba 100644
 --- a/Makefile
 +++ b/Makefile
-@@ -41,7 +41,7 @@ PLATS= aix bsd c89 freebsd generic linux macosx mingw posix solaris
+@@ -41,7 +41,7 @@ PLATS= guess aix bsd c89 freebsd generic linux linux-readline macosx mingw posix
```

Comment:
I think you have mistakenly removed `solaris` here.

Reply:
This is actually generated, not done by hand.

Reply:
I dont know why it dropped the `solaris` in the end, but it would apply into the diff though, thanks for catching this out. :)

Reply:
You're welcome @chenrui333 -- I was working on this myself earlier today, but you beat me to it.

Reply:
Looks like `solaris` is still missing here - does it need to be added back?